examples/request_version/request_version.cc

```diff
@@ -0,0 +1,137 @@
+/**************************************************************************/ /**
+* @brief Message encode example.
```

Comment:
Update this to explain what this example app does, and also add the app to https://github.com/PointOneNav/fusion-engine-client/tree/master#example-applications